### ISSUE TITLE
App fails to render

### ISSUE THREAD

**Issue by flrs-edf** [IEA-Task-43/digital_wra_data_standard#2]
The app fails to render when reaching the "measurement point" details. I suspect the error comes from the schema only allowing null values for `mounting_arrangement` (see [here](https://github.com/IEA-Task-43/digital_wra_data_standard/blob/dc233ddc37e8930a873ba9a26e592bc3447aef47/schema/iea43_wra_data_model.schema.json#L330)).

This is what I am getting in the console when trying to add a measurement point:

```
react-dom.production.min.js:4636 Error: Unknown type: {"type":"null"}
    at e (uischema.js:195)
    at Object.t.generateDefaultUISchema [as uiSchema] (uischema.js:219)
    at Object.t.findUISchema (index.js:145)
    at combinators.js:66
    at Array.map (<anonymous>)
    at Object.t.createCombinatorRenderInfos (combinators.js:63)
    at MaterialAnyOfRenderer.js:66
    at ho (react-dom.production.min.js:3610)
    at Ps (react-dom.production.min.js:6612)
    at Au (react-dom.production.min.js:5648)
vs @ react-dom.production.min.js:4636
r.callback @ react-dom.production.min.js:5081
Si @ react-dom.production.min.js:3059
Ei @ react-dom.production.min.js:3047
Ru @ react-dom.production.min.js:6345
t.unstable_runWithPriority @ scheduler.production.min.js:270
Ja @ react-dom.production.min.js:2794
Fu @ react-dom.production.min.js:6105
wu @ react-dom.production.min.js:5408
(anonymous) @ react-dom.production.min.js:2829
t.unstable_runWithPriority @ scheduler.production.min.js:270
Ja @ react-dom.production.min.js:2794
Qa @ react-dom.production.min.js:2824
Ga @ react-dom.production.min.js:2814
se @ react-dom.production.min.js:7244
_r @ react-dom.production.min.js:1710
uischema.js:195 Uncaught Error: Unknown type: {"type":"null"}
    at e (uischema.js:195)
    at Object.t.generateDefaultUISchema [as uiSchema] (uischema.js:219)
    at Object.t.findUISchema (index.js:145)
    at combinators.js:66
    at Array.map (<anonymous>)
    at Object.t.createCombinatorRenderInfos (combinators.js:63)
    at MaterialAnyOfRenderer.js:66
    at ho (react-dom.production.min.js:3610)
    at Ps (react-dom.production.min.js:6612)
    at Au (react-dom.production.min.js:5648)
e @ uischema.js:195
t.generateDefaultUISchema @ uischema.js:219
t.findUISchema @ index.js:145
(anonymous) @ combinators.js:66
t.createCombinatorRenderInfos @ combinators.js:63
(anonymous) @ MaterialAnyOfRenderer.js:66
ho @ react-dom.production.min.js:3610
Ps @ react-dom.production.min.js:6612
Au @ react-dom.production.min.js:5648
ju @ react-dom.production.min.js:5637
wu @ react-dom.production.min.js:5393
(anonymous) @ react-dom.production.min.js:2829
t.unstable_runWithPriority @ scheduler.production.min.js:270
Ja @ react-dom.production.min.js:2794
Qa @ react-dom.production.min.js:2824
Ga @ react-dom.production.min.js:2814
se @ react-dom.production.min.js:7244
_r @ react-dom.production.min.js:1710
```

**Comment by stephenholleran** [comment]
I have removed `anyOf` which included that `type: null`. Not sure if that fixes it. 

I have added a lot since also. And still got 'mast_properties' to add yet.

**Comment by flrs-edf** [comment]
This may have been fixed by f80b6ea7c4d4646b9fe4f1d7e9242e5e72f92759 and bd31397730736738002eb95606e3ddb0b2777466. The app seems to work now.
